tests/contrib/timeseries/test_gp.py

```diff
@@ -2,51 +2,73 @@
 
 from tests.common import assert_equal
 import pyro
-from pyro.contrib.timeseries import IndependentMaternGP
+from pyro.contrib.timeseries import IndependentMaternGP, LinearlyCoupledMaternGP
 import pytest
 
 
+@pytest.mark.parametrize('model', ['lcmgp', 'imgp'])
 @pytest.mark.parametrize('nu', [1.5, 2.5])
 @pytest.mark.parametrize('obs_dim', [1, 3])
 @pytest.mark.parametrize('T', [11, 37])
-def test_independent_matern_gp(nu, obs_dim, T):
+def test_independent_matern_gp(model, nu, obs_dim, T):
     torch.set_default_tensor_type('torch.DoubleTensor')
     dt = 0.1 + torch.rand(1).item()
 
-    gp = IndependentMaternGP(nu=nu, obs_dim=obs_dim, dt=dt,
-                             log_length_scale_init=torch.randn(obs_dim),
-                             log_kernel_scale_init=torch.randn(obs_dim),
-                             log_obs_noise_scale_init=torch.randn(obs_dim))
+    if model == 'lcmgp':
+        if obs_dim == 1:
```

Comment:
lcmgp expects obs_dim > 1

Reply:
nit: could you either `pytest.skipif(obs_dim == 1)` or winnow the parameters as
```py
@pytest.mark.parametrize('model,obs_dim', [('lcmgp', 3),
                                           ('imgp', 1),
                                           ('imgp', 3)])
```

Reply:
done